vhd/packer/configure-windows-vhd.ps1

```diff
@@ -176,7 +176,7 @@ function Install-ContainerD {
 }
 
 function Install-Docker {
-    $defaultDockerVersion = "20.10.5"
+    $defaultDockerVersion = "20.10.7"

```

Comment:
Windows releases are via Mirantis Docker EE which can be behind the Linux moby releases and sometimes skip patch releases.  Currently the latest release version is `20.10.6`

Reply:
you can find latest from the link at: https://dockermsft.azureedge.net/dockercontainer/DockerMsftIndex.json

Reply:
Got it, thanks!